packages/core/src/components/Tooltip/index.tsx

```diff
@@ -86,6 +86,13 @@ export class Tooltip extends React.Component<Props & WithStylesProps, State> {
     return null;
   }
 
+  componentDidMount() {
+    // eslint-disable-next-line
```

Comment:
Add the name of the rule this is disabling.

Reply:
updated!